lib/Mojolicious/Command/generate/app.pm

```diff
@@ -202,3 +211,12 @@ and the layout "templates/layouts/default.html.ep",
 <%%= link_to 'here' => '/index.html' %> to move forward to a static page. To
 learn more, you can also browse through the documentation
 <%%= link_to 'here' => '/perldoc' %>.
+
+@@ config
+{
+  db => {
+      dsn => "...",
+      username => "...",
+      password => "...",
```

Comment:
That's not how we configure database connections in the Mojolicious world. (https://github.com/kraih/mojo-pg/blob/master/examples/blog/blog.conf)